core_data_modules/cleaners/somali/demographic_cleaner.py

```diff
@@ -130,3 +131,134 @@ def clean_age_within_range(age, min_age_inclusive=10, max_age_inclusive=99):
             return Codes.NOT_CODED
 
         return age
+
+    @staticmethod
+    def get_district(location):
```

Comment:
Are these really cleaners? They seem to be building a lookup dictionary which could be of use for other purposes than cleaning?

Reply:
Will move these in about 2 PRs' time (https://github.com/AfricasVoices/CoreDataModules/issues/71)